kernel/packages/shared/apis/RestrictedActions.ts

```diff
@@ -9,43 +9,14 @@ import { lastPlayerPosition } from '../world/positionThings'
 import { browserInterface } from "../../unity-interface/BrowserInterface"
 
 export enum Permission {
-  ALLOW_TO_MOVE_PLAYER_INSIDE_SCENE = 'ALLOW_TO_MOVE_PLAYER_INSIDE_SCENE'
+  ALLOW_TO_MOVE_PLAYER_INSIDE_SCENE = 'ALLOW_TO_MOVE_PLAYER_INSIDE_SCENE',
+  ALLOW_TO_TRIGGER_AVATAR_EMOTE = 'ALLOW_TO_TRIGGER_AVATAR_EMOTE'
 }
 
-export interface IRestrictedActionModule {
-  movePlayerTo(newPosition: Vector3, cameraTarget?: Vector3): Promise<void>
-}
-
-@registerAPI('RestrictedActionModule')
```

Comment:
Renaming `RestrictedActions` to `RestrictedActionModule` will break every scene currently using `movePlayerTo(position: PositionType, cameraTarget?: PositionType)` (moved to the module)

Reply:
Good catch! Just pushed the fix